src/modules/git_commit.rs

```diff
@@ -51,6 +51,9 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
 }
 
 fn git_tag(repo: &Repo, config: &GitCommitConfig) -> Option<String> {
+    if config.tag_disabled {
+        return None;
+    }
```

Comment:
I would prefer doing the check in `map` as `"tag" if !config.tag_disabled => `

Reply:
Thanks for you suggestion. Updated.